zgw/objecten-api/src/test/kotlin/com/ritense/objectenapi/service/ZaakObjectServiceTest.kt

```diff
@@ -50,7 +51,8 @@ internal class ZaakObjectServiceTest {
     val zaakInstanceLinkService = mock<ZaakInstanceLinkService>()
     val pluginService = mock<PluginService>()
     val formDefinitionService = mock<FormDefinitionService>()
-    val zaakObjectService = ZaakObjectService(zaakInstanceLinkService, pluginService, formDefinitionService)
+    val objectManagementInfoProvider = mock<ObjectManagementInfoProvider>()
+    val zaakObjectService = ZaakObjectService(zaakInstanceLinkService, pluginService, formDefinitionService, objectManagementInfoProvider)
```

Comment:
This stuff would typically be done in an init method using the `BeforeEach` method. Do you mind moving the initialization there?